Aztec/Classes/EditorView/EditorView.swift

```diff
@@ -6,12 +6,14 @@ import UIKit
 public class EditorView: UIView {
     public let htmlTextView: UITextView
     public let richTextView: TextView
+    public let htmlStorage: HTMLStorage
```

Comment:
Do you think it would be better to have a computed property instead? Something like

```swift
public var htmlStorage: HTMLStorage {
    guard let htmlStorage = htmlTextView.textStorage as? HTMLStorage else {
        fatalError("If this happens, something is very off on the init config")
    }
    return htmlStorage
}
```

I don't particularly like the `fatalError`, even though the guard should always succeed. Alternatively we can make it an optional, but that wouldn't make much sense for the "user", and how to handle the hypothetical `nil` case?

But it would remove the hassle with the coder init.

None look like a great option to me really, so I'm happy to keep it as it is if you think it is the best one 🙂

Reply:
Yep I like it!